src/audio/multiband_drc/multiband_drc.c

```diff
@@ -205,10 +210,8 @@ static int multiband_drc_setup(struct processing_module *mod, int16_t channels,
 
 	/* Setup Crossover, Emphasis EQ, Deemphasis EQ, and DRC */
 	ret = multiband_drc_init_coef(mod, channels, rate);
-	if (ret < 0)
-		return ret;
-
-	return 0;
+	/* This will return 0 if no error occurred, or the negative error code. */
+	return ret;
```

Comment:
if you're doing this, you could as well just make it
```
multiband_drc_init_coef(mod, channels, rate);
```
and the comment about the error code isn't needed - we have many functions in SOF, returning error codes, we don't need to explain this in each of them.